contracts/protocol/pool/DefaultReserveInterestRateStrategy.sol

```diff
@@ -5,9 +5,10 @@ import {IERC20} from '../../dependencies/openzeppelin/contracts/IERC20.sol';
 import {WadRayMath} from '../libraries/math/WadRayMath.sol';
 import {PercentageMath} from '../libraries/math/PercentageMath.sol';
 import {DataTypes} from '../libraries/types/DataTypes.sol';
-import {IReserveInterestRateStrategy} from '../../interfaces/IReserveInterestRateStrategy.sol';
 import {IPoolAddressesProvider} from '../../interfaces/IPoolAddressesProvider.sol';
 import {Errors} from '../libraries/helpers/Errors.sol';
+import {IDefaultInterestRateStrategy} from "../../interfaces/IDefaultInterestRateStrategy.sol";
+import {IReserveInterestRateStrategy} from "../../interfaces/IReserveInterestRateStrategy.sol";
```

Comment:
seems not needed

Reply:
This is needed for the inherit docs

Reply:
https://github.com/aave/aave-v3-core/pull/763/files#diff-1f7acd1ca2faeee390ca9f1bde1da92aa4397a3307a8f43f2b651f8e01dc5a0cR158